sshmuxer/tcphandler.go

```diff
@@ -25,6 +25,10 @@ func handleTCPListener(check *channelForwardMsg, bindPort uint32, requestMessage
 		return nil, nil, "", "", fmt.Errorf("Error assigning requested port to tunnel")
 	}
 
+	if tcpPort == 0 {
```

Comment:
`tcpPort` is 0 if a user requests a random port. We don't want to actually error on this.

Reply:
Sorry, wrong reading